cmd/up/up.go

```diff
@@ -719,14 +719,25 @@ func (up *upContext) startSyncthing(ctx context.Context) error {
 	defer spinner.Stop()
 
 	if err := up.Sy.Run(ctx); err != nil {
-		return err
+		log.Infof("local synchronization process is unresponsive: %s", err.Error())
+		if up.Dev.Interface == model.Localhost {
+			return errors.UserError{
+				E:    fmt.Errorf("Local synchronization process is unresponsive on localhost"),
+				Hint: "Could you try adding 'interface: 0.0.0.0' to your okteto manifest?",
```

Comment:
Is the `syncthing` instance we start listening on `localhost` or `0.0.0.0`?  I don't think this is a good hint for a connectivity issue.